docs/guides/quickfort-alias-guide.rst

```diff
@@ -0,0 +1,850 @@
+.. _quickfort-alias-guide:
+
+Quickfort Alias Guide
+=====================
+
+Aliases allow you to use simple words to represent complicated key sequences
+when configuring buildings and stockpiles in quickfort ``#query`` blueprints.
+
+For example, say you have the following ``#build`` and ``#place`` blueprints:
+
+::
+
+    #build masonry workshop
+    ~, ~,~,`,`,`
+    ~,wm,~,`,`,`
+    ~, ~,~,`,`,`
+
+    #place stockpile for mason
+    ~,~,~,s,s,s
+    ~,~,~,s,s,s
+    ~,~,~,s,s,s
+
+and you want to configure the stockpile to hold only non-economic ("other")
+stone and to give to the adjacent mason workshop. You could write the
+key sequences directly:
+
+::
+
+    #query configure stockpile with expanded key sequences
+    ~,~,~,s{Down 5}deb{Right}{Down 2}p^,`,`
+    ~,~,~,g{Left 2}&,                   `,`
+    ~,~,~,`,                            `,`
+
+or you could use aliases:
+
+::
+
+    #query configure stockpile with aliases
+    ~,~,~,otherstone,`,`
+    ~,~,~,give2left, `,`
+    ~,~,~,`,         `,`
+
+If the stockpile had only a single tile, you could also replay both aliases in
+a single cell:
+
+::
+
+    #query configure mason with multiple aliases in one cell
+    ~,~,~,{otherstone}{give2left},`,`
+    ~,~,~,`,                      `,`
+    ~,~,~,`,                      `,`
+
+With aliases, blueprints are much easier to read and understand. They also
+save you from having to copy the same long key sequences everywhere.
+
+Alias definition files
+----------------------
+
+DFHack comes with a library of aliases for you to use that are always
+available when you run a ``#query`` blueprint. Many blueprints can be built
+with just those aliases. This "standard alias library" is stored in
+:source:`data/quickfort/aliases-common.txt`. The aliases in that file are
```

Comment:
Minor: may be worth clarifying that this is `hack/data/...` at runtime (since DF has its own top-level `data` folder). If it's intended to be read-only, this is probably not as important, but you could add something like "in the `hack` folder" if you want to avoid duplicating the file path.

Reply:
done